swatch-core/src/test/resources/test-clowder-config.json

```diff
@@ -31,7 +31,19 @@
         "brokers": [
             {
                 "hostname": "env-rhsm-kafka.rhsm.svc",
-                "port": 29092
+                "port": 29092,
+                "cacert": "Dummy value",
+                "authtype": "sasl",
+                "sasl": {
+                    "username": "john",
+                    "password": "doe",
+                    "securityProtocol":"SASL_SSL",
+                    "saslMechanism":"PLAIN"
+                }
+            },
+            {
+                "hostname": "env-rhsm-kafka-secondary.rhsm.svc",
+                "port": 29093
             }
```

Comment:
Is this what the multi-broker configuration looks like in practice?  And if so, my reading of the code is that the SASL bit from the first item in the list will be applied to the second item in the list.  That's fine, but I just want to have some comments to that effect in the PropertySource class.

Reply:
Added a comment in ClowderJsonPathPropertySource.KAFKA_BROKERS_PROPERTIES.